packages/data-api/src/TupaiaDataApi.js

```diff
@@ -55,21 +56,23 @@ export class TupaiaDataApi {
     }));
   }
 
-  async fetchDataElements(dataElementCodes) {
+  async fetchDataElements(dataElementCodes, { includeOptions = true }) {
     if (!dataElementCodes || !Array.isArray(dataElementCodes)) {
       throw new Error('Please provide an array of data element codes');
     }
-    return new SqlQuery(
+    const sqlQuery = new SqlQuery(
       `
-      SELECT code, name
+      SELECT code, name, type
```

Comment:
Does this need to fetch `options` as well?

Reply:
oops...